tests/subsys/zigbee/osif/crypto/CMakeLists.txt

```diff
@@ -6,19 +6,23 @@
 
 cmake_minimum_required(VERSION 3.13.1)
 
-add_definitions(-Wno-packed-bitfield-compat)
-
 find_package(Zephyr HINTS $ENV{ZEPHYR_BASE})
 
-zephyr_compile_definitions(CONFIG_ZBOSS_OSIF_LOG_LEVEL=LOG_LEVEL_DBG)
+target_compile_options(app PRIVATE -Wno-packed-bitfield-compat)
+
+target_compile_definitions(app PRIVATE
+  CONFIG_ZBOSS_OSIF_LOG_LEVEL=LOG_LEVEL_DBG
```

Comment:
Why do you set Kconfig values like this, instead of adding an entry in `prj.conf` file? I have missed it in the previous PR.

Reply:
This test verifies the crypto component only and zigbee subsys remains inactive.
`CONFIG_ZBOSS_OSIF_LOG_LEVEL` depends on `CONFIG_ZIGBEE` which is not set.

Reply:
OK, understood.